pkg/controllers/disruption/orchestration/queue.go

```diff
@@ -202,10 +204,11 @@ func (q *Queue) Reconcile(ctx context.Context, _ reconcile.Request) (reconcile.R
 		// Log the error
 		logging.FromContext(ctx).With("nodes", strings.Join(lo.Map(cmd.candidates, func(s *state.StateNode, _ int) string {
 			return s.Name()
-		}), ",")).Errorf("failed to disrupt nodes, %s", multiErr)
+		}), ","), "uid", cmd.uid).Errorf("failed to disrupt nodes, %s", multiErr)
```

Comment:
```suggestion
		}), ","), "reconcile-id", cmd.uid).Errorf("failed to disrupt nodes, %s", multiErr)
```

Is this value really just a reconciliation id or we could even brand it as the disruption-id?

Reply:
I'm not a fan of reconcile-id, as it insinuates it's the same reconcile. Which do you prefer?

Reply:
synced offline, went with command-uid